etcdserver/api/v3rpc/watch.go

```diff
@@ -76,6 +80,13 @@ func (sws *serverWatchStream) recvLoop() error {
 			return err
 		}
 
+		rh := &pb.ResponseHeader{
```

Comment:
can avoid some repetition if this is `sws.newResponseHeader() *pb.ResponseHeader` and using it like `Header : sws.newResponseHeader()`. up to you though